cliff.toml

```
@@ -0,0 +1,94 @@
+# git-cliff configuration file
+# Documentation: https://git-cliff.org/docs/configuration
+
+[git]
+# parse the commits based on https://www.conventionalcommits.org
+conventional_commits = true
+# filter out the commits that are not conventional
+filter_unconventional = true
+# process each line of a commit as an individual commit
+split_commits = false
+# regex for preprocessing the commit messages
+commit_preprocessors = [
+  { pattern = '#(\d+)', replace = "[#${1}](https://github.com/deltachat/deltachat-desktop/issues/${1})" },
+]
+# regex for parsing and grouping commits
+commit_parsers = [
+    { message = "^feat", group = "Features / Changes"},
+    { message = "^fix", group = "Fixes"},
+    { message = "^refactor", group = "Refactor"},
+    { message = "^perf", group = "Performance"},
+    { message = "^test", group = "Tests"},
+    { message = "^style", group = "Styling"},
+    { message = "^chore\\(release\\): prepare for", skip = true},
+    { message = "^chore", group = "Miscellaneous Tasks"},
+    { message = "^build", group = "Build system"},
+    { message = "^docs", group = "Documentation"},
+    { message = "^ci", group = "CI"},
+    { message = ".*", group = "Other"},
```

```suggestion
    { message = "^feat", group = "Features / Changes"},
    { message = "^fix", group = "Fixes"},
    { message = "^refactor", group = "Refactor", skip = true},
    { message = "^perf", group = "Performance"},
    { message = "^test", group = "Tests", skip = true},
    { message = "^style", group = "Styling"},
    { message = "^chore\\(release\\): prepare for", skip = true},
    { message = "^chore", group = "Miscellaneous Tasks", skip = true},
    { message = "^build", group = "Build system", skip = true},
    { message = "^docs", group = "Documentation"},
    { message = "^ci", group = "CI", skip = true},
    { message = ".*", group = "Other"},
```

Users don't need to know about refactors and stuff.